### PR TITLE
added summary for 2021-04 uPSC meeting

### PR DESCRIPTION
**Pull request by jonathanmtran**

### CODE REVIEW

File: governance/meetings/uportal-steering-committee-2021-04-14.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+## Attendees
+
+Jim Helwig, Christian Cousquer, Jonathan Tran, Toben Archer, Benito Gonzalez, and Julien Gribonvald
+
+## Open Apereo 21
+
+The steering committee discussed community meeting/workshops for Open Apereo 21. The meetings/workshops include:
+
+* Roadmap (meeting)
+* First contribution workshop
```
Comment (jimhelwig) [inline governance/meetings/uportal-steering-committee-2021-04-14.md:12]: Consider adding the following:
```suggestion
* First contribution workshop

## uPSC task tracking

The steering committee discussed current efforts at using Github for prioritizing and tracking committee tasks.
```

**Review by jimhelwig — APPROVED** [review]
I offered up a suggestion that could be considered before merging.

**Review by ChristianMurphy — APPROVED** [review]

**Comment by jonathanmtran** [comment]
@jimhelwig I'm good with the suggestion. Do you have the option to commit it? That way your great suggestions are yours :)

**Comment by ChristianMurphy** [comment]
If you use the GitHub UI to apply it, GitHub has a neat trick that it will automatically add the suggester as a co-committer.

**Pull request merged**